### DISCUSSION TITLE
Help with json flow in nodered to send snapshot

### DISCUSSION THREAD

**Discussion by Yonny24** [blakeblackshear/frigate#2321] [Ask A Question]
I'm really struggling to encode the mqtt snapshot data to an image and send through telegram.
I have the config set up to send via 'frigate/camera/object/snapshot'
But then I'm stuck. Tried base64 node and other types but all I get is string of characters so I'm obviously going wrong.

Please can someone share their json and/or flows please?

Many thanks

**Comment by simeon-walker** [comment]
I have an mqtt in node subscribed to the snapshot topic, into a function that does: 
````
var tmsg = {
    "payload": {
        "content": msg.payload,
        "type": "photo",
        "chatId": "-12345"
    }
}
return tmsg;
````
then into the Telegram sender.

Hope this helps, if you need more details remind me tomorrow, bit late for me now!
